modules/standard/Map.chpl

```diff
@@ -315,38 +314,36 @@ module Map {
      */
     proc getReference(k: keyType) ref
```

Comment:
I think that with `chpl__hashtable` as the implementation, it should be possible to remove the where clause. More broadly, I think that with the use of `chpl__hashtable` it should be possible to remove (or at least deprecate) all the workarounds we added for non-nilable classes due to associative arrays.

Reply:
We can deprecate/remove in a followup effort after discussion, though.